public/app/plugins/panel/candlestick/fields.test.ts

```diff
@@ -1,12 +1,12 @@
 import { createTheme, toDataFrame } from '@grafana/data';
 
 import { prepareCandlestickFields } from './fields';
-import { CandlestickOptions, VizDisplayMode } from './models.gen';
+import { Options, VizDisplayMode } from './types';
 
 const theme = createTheme();
 
 describe('Candlestick data', () => {
-  const options: CandlestickOptions = {} as CandlestickOptions;
+  const options: Options = {} as Options;
```

Comment:
```suggestion
  const options = {} as Options;
```
These types are redundant since we're using type assertions.